corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -161,6 +169,14 @@ hqDefine("data_dictionary/js/data_dictionary", [
             return self.dataType() === 'select';
         });
 
+        self.confirmDeleteProperty = function () {
+            const $modal = $("#delete-case-prop-modal").modal('show');
+            $("#delete-case-prop-btn").off("click").on("click", () => {
```

Comment:
Interesting. Why do we unbind the click event before binding it?

Reply:
An element can have more than one event binding at a time. This is also true if it's the same event being binded multiple times. To prevent duplicate events getting triggered, we therefore need to unbind the old event first before adding it again.